asym_crypto_yaml/crypto.py

```diff
@@ -417,7 +417,7 @@ def generate_public_key_to_file(private_key_file_path, public_key_file_output_pa
 def encrypt_value_and_print(unencrypted_value, public_key_file):
     public_key = load_public_key_from_file(public_key_file)
     encrypted_value = encrypt_value(unencrypted_value, public_key)
-    print(_safe_dump(encrypted_value))
+    print((_safe_dump(encrypted_value)))
```

Comment:
Is this change necessary? Shouldn't single braces suffice?

Reply:
reverted.